package.json

```diff
@@ -1802,16 +1803,16 @@
                     "type": "boolean",
                     "default": false,
                     "description": "Experimental feature to enable execution analysis in notebooks",
-                    "scope": "application"
+                    "scope": "application",
```

Comment:
Why are the settings `executionAnalysis.enabled` and `interactive.notebookRepl`  using different tags.
One has onExp and other doesn't have it

Reply:
onExP means the setting could be assigned to an A/B experiment.
If that's not the intended meaning, I could remove that tag from the setting.

Reply:
is there an A/B experiment for (or planned for) `executionAnalysis.enabled`? that's what that tag is for.
Eventually we'll get an experiment for the notebook REPL.